src/redux/books/bookSlice.js

```diff
@@ -1,27 +1,70 @@
-import { createSlice } from '@reduxjs/toolkit';
-import booksList from './booksList';
+import { createSlice, nanoid, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
 
-const initialState = booksList;
+const URL_FETCH_BOOKS = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/Lz0Xg73BN9mDXt1XFhvh/books';
+const URL_BOOKS = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/Lz0Xg73BN9mDXt1XFhvh/books';
```

Comment:
[OPTIONAL]
- [ ] `URL_FETCH_BOOKS` and `URL_BOOKS` are quite the same, you may consider deleting one for clean code best practice.